packages/pilot/src/pages/LoggedArea/Sidebar.js

```diff
@@ -52,7 +53,12 @@ const Sidebar = ({
     balance={balance}
     companyName={companyName}
     links={values(routes)
-      .filter(({ hidden }) => !hidden)
+      .filter(({ hidden }) => {
+        if (typeof hidden === 'function') {
```

Comment:
Não gosto de um prop poder ser tanto um boleano como uma função. Acho que ficaria melhor com um filter a mais e uma prop separada com a função de filtrar por flags (capabilities).

Reply:
Talvez possamos mudar o `hidden` para uma função separada `validateVisibility` a qual quando existir vai ter prioridade sobre o `hidden` ex:
```suggestion
        .filter(({ hidden, validateVisibility }) => {
        if (validateVisibility) {
           return validateVisibility(companyCapabilities)
```